### PR TITLE
Fixed sublanding duplicate entries

### PR DESCRIPTION
**Pull request by kave**
Sublanding pages were incorrectly showing elements due to multiple for loops

## Additions

-

## Removals

-

## Changes

- removed duplicate for loops

## Testing

- Create and preview a sublanding page. Should only show elements once

## Review

- @kurtw 
- @richaagarwal 

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/sublanding-page/index.html [hunk 1]
```diff
@@ -62,26 +50,22 @@
 
 {% block content_sidebar scoped -%}
     {% for block in page.sidefoot %}
-        {% if 'related_posts' in block.block_type %}
-            {{ related_posts.render(block) }}
-        {% elif 'email_signup' in block.block_type %}
-            {{ render_stream_child(block) }}
-        {% else %}
-            {% if 'heading' in block.block_type %}
+        <div class="block block__flush-top">
```
Comment (richaagarwal) [inline cfgov/jinja2/v1/sublanding-page/index.html]: what does wrapping it in this div do?

**Comment by kurtw** [comment]
gtm

File: cfgov/jinja2/v1/sublanding-page/index.html (hunk 1 above)
Comment (kave) [inline cfgov/jinja2/v1/sublanding-page/index.html]: @richaagarwal i noticed all our sidefoot blocks in other templates added this div class `<div class="block block__flush-top">` to resolve spacing issues. noticed it was missing here, so included it in

**Comment by richaagarwal** [comment]
:+1:

**Comment by kave** [comment]
Travis error associated with Coverrals integration

**Pull request merged**